casepro/statistics/models.py

```diff
@@ -249,22 +252,73 @@ def render_book(self, book):
                 self.write_row(sheet, row, [day] + totals)
                 row += 1
 
+        elif self.type == self.TYPE_USER:
+            replies_sheet = book.add_sheet(six.text_type(_("Replies Sent")))
+            cases_opened_sheet = book.add_sheet(six.text_type(_("Cases Opened")))
+            cases_closed_sheet = book.add_sheet(six.text_type(_("Cases Closed")))
+
+            users = self.org.get_org_users().order_by('pk')
```

Comment:
Also if the user is looking at non-partner users, this export should reflect that.